views/views.go

```diff
@@ -147,5 +203,15 @@ func New(conf Config) *Views {
 	// Struct validator
 	v.validate = validator.New()
 
-	return &v
+	return v
+}
+
+func (v *Views) errorHandle(c echo.Context, err error) error {
+	data := struct {
+		Error string
+	}{
+		Error: err.Error(),
+	}
+	log.Println(data.Error)
+	return v.template.RenderNoNavsTemplate(c.Response().Writer, data, templates.ErrorTemplate)
```

Comment:
This is not actually necessary in Echo - the general pattern is to return err (bear with me), and then handle it in a top-level error handler - [echo.labstack.com/docs/error-handling](https://echo.labstack.com/docs/error-handling)

Reply:
I think I used a custom error page because it was easier to do it that way not to just have a JSON response but will investigate in a bit

Reply:
You can still have custom error pages with Echo's error handler - set `echo.HTTPErrorHandler` to a function that renders out the template of your choosing.

Reply:
Tomorrow problem

Reply:
Fixed